docs/scripts/register.js

```diff
@@ -1,4 +1,4 @@
-import {plugins} from 'chart.js';
+import Chart from 'chart.js/auto';
 import plugin from '../../dist/chartjs-plugin-deferred.js';
 
-plugins.register(plugin);
+Chart.register(plugin);
```

Comment:
I think I prefer [this way](https://github.com/chartjs/chartjs-plugin-datalabels/blob/master/docs/samples/register.js#L1) of registering controllers, elements, etc. 

```js
import {Chart, registerables} from 'chart.js';
import plugin from '../../dist/chartjs-plugin-deferred.js';

Chart.register(...registerables);
Chart.register(plugin);
```